### DISCUSSION TITLE
rename or spoiler command

### DISCUSSION THREAD

**Discussion by Gam3rMom3nt** [machineonamission/mediaforge#25] [Ideas]
I just think a spoiler or rename command would be good considering how bad changing file names on mobile is.

edit: I don't know how to close a discussion

**Comment by machineonamission** [comment]
the problem is that filenames dont always persist when saved, like IOS discord renames all files to image0 or image1

**Reply by Gam3rMom3nt** [reply]
okay cool

**Comment by machineonamission** [comment]
added in 6ba516a
